composer.json

```
@@ -17,8 +17,8 @@
   "require": {
     "php": ">=7.4|>=8.0",
     "ext-json": "*",
-    "nikic/php-parser": "^4.4",
-    "jeremeamia/functionparser": "^1.0"
+    "nikic/php-parser": "^5.0",
+    "simpletools/functionparser": "1.0.1"
```

```suggestion
    "simpletools/functionparser": "^1.0.1"
```

Locking it to a specific version is usually bad form in a library, ^ allows future backwards-compatible versions of this library to be used as well.

---

I have amended the version to `"simpletools/functionparser": "^1.0"` on the pull branch now.